index.html

```diff
@@ -110,17 +110,17 @@ <h3>Curl from Google Chrome</h3>
     </div>
     <div class="row border-top pt-4">
       <div class="col-sm-4">
-        <div class="row pb-3">
-          <h3>Support Us</h3>
-          <div class="well larger-text">
-            <p>GitHub is matching all contributions to this project on <a href="https://github.com/sponsors/NickCarneiro">Github Sponsors</a>.
-            This is a surprisingly good deal and I'm not sure how long it's going to last.
```

Comment:
I removed this sentence because [you can't sign up for matching anymore](https://docs.github.com/en/sponsors/getting-started-with-github-sponsors/about-github-sponsors#about-the-github-sponsors-matching-fund) and I think it'll last indefinitely for everyone who has